tests/rt.conf

```diff
@@ -133,8 +133,9 @@ RUN     | fv3_gfs_v16_ras_debug
 # CPLD tests                                                                                                                                                                      #
 ###################################################################################################################################################################################
 
-COMPILE | APP=S2S SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled             | - wcoss_cray jet.intel                  | fv3 |
+COMPILE | APP=S2S SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled,FV3_GFS_v15p2_couplednsst             | - wcoss_cray jet.intel                  | fv3 |
```

Comment:
Would you please add a GFSv16 coupled case as coupled P6 already moved to GFSv16_coupled? We may remove the old configures soon.

Reply:
You mean to add one ccpp suite, like FV3_GFS_v16_couplednsst, since FV3_GFS_v15p2_coupled will be removed soon. 

If yes, I can create a ccpp suite, FV3_GFS_v16_couplednsst, based on FV3_GFS_v16_coupled, as I did for FV3_GFS_v15p2_couplednsst. But a test is required to deliver FV3_GFS_v16_couplednsst.

Reply:
> @XuLi-NOAA Please close PR #453 since it has been replaced by PR #483.

PR #453 closed since it has been replaced by # 483